From d1b65d4de6305c7f4c3987ddc6541f51d59061d6 Mon Sep 17 00:00:00 2001
From: Loren Huang <b02279@freescale.com>
Date: Mon, 3 Sep 2012 09:45:04 +0800
Subject: ENGR00222148 [MX6SL]Shrink GPU reserved memory to 32M

MX6SL only has 512M memory.Shrinking GPU reserved memory to 32M
can help out of memory issue on MX6SL.This patch will increase
96M system memory, so it will help the case which requests lots
of system memory.Like multiple application running, etc.

And MX6SL doesn't have 3D, 32M is recommended by vivante.

Signed-off-by: Loren Huang <b02279@freescale.com>
---
 arch/arm/mach-mx6/board-mx6sl_arm2.c |    2 +-
 arch/arm/mach-mx6/board-mx6sl_evk.c  |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6sl_arm2.c b/arch/arm/mach-mx6/board-mx6sl_arm2.c
index 4302b68..477ea8c 100755
--- a/arch/arm/mach-mx6/board-mx6sl_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6sl_arm2.c
@@ -610,7 +610,7 @@ static struct mxc_dvfs_platform_data mx6sl_arm2_dvfscore_data = {
 };
 
 static struct viv_gpu_platform_data imx6q_gpu_pdata __initdata = {
-	.reserved_mem_size = SZ_128M,
+	.reserved_mem_size = SZ_32M,
 };
 
 void __init early_console_setup(unsigned long base, struct clk *clk);
diff --git a/arch/arm/mach-mx6/board-mx6sl_evk.c b/arch/arm/mach-mx6/board-mx6sl_evk.c
index 81654a0..23fcff5 100644
--- a/arch/arm/mach-mx6/board-mx6sl_evk.c
+++ b/arch/arm/mach-mx6/board-mx6sl_evk.c
@@ -623,7 +623,7 @@ static struct mxc_dvfs_platform_data mx6sl_evk_dvfscore_data = {
 };
 
 static struct viv_gpu_platform_data imx6q_gpu_pdata __initdata = {
-	.reserved_mem_size = SZ_128M,
+	.reserved_mem_size = SZ_32M,
 };
 
 void __init early_console_setup(unsigned long base, struct clk *clk);
-- 
1.7.9.5

